### PR TITLE
Update webhook test image to multi-arch

### PR DESCRIPTION
**Pull request by dims**
**What this PR does / why we need it**:

Use the `image-util.sh` to build the container image so we can generate the full manifest for multi-arch images 

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

**Special notes for your reviewer**:
/cc @mkumatag 
/cc @luxas 

**Release note**:

```release-note
NONE
```

### CODE REVIEW

**Comment by dims** [comment]
related to https://github.com/kubernetes/kubernetes/issues/66626

File: test/images/webhook/Dockerfile [hunk 1]
```diff
@@ -12,7 +12,7 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-FROM alpine:latest
+FROM BASEIMAGE
```
Comment (spiffxp) [inline test/images/webhook/Dockerfile]: I was going to nit on BASEIMAGE making this an incomplete `Dockerfile`, in which case I'd want to rename it to `Dockerfile.in`,`Dockerfile.build` or something... but I guess this is the pattern used all throughout test/images already.  So.. carry on.

Anyone know if there's an issue for this?

Comment (luxas) [reply test/images/webhook/Dockerfile]: No I don't see that huge problem with it either. We use  `make build` consistently instead to put all other prerequisites in place as well, even though the Dockerfile wouldn't be a template.

**Review by luxas — APPROVED** [review]
/approve
nit on the version string but it was that so I'm good I guess

**Pull request merged**